src/App.jsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 
+import ListContainer from './ListContainer';
+import FormContainer from './FormContainer';
+
 export default function App() {
   return (
     <div>
-      Hello
+      <h1>Restaurants</h1>
+      <ListContainer />
+      <FormContainer />
```

Comment:
```suggestion
      <RestaurantsContainer />
      <RestaurantCreateContainer />
```

위와 같이 의미를 좀 더 드러내길 권장합니다.

TextInputField처럼 아주 일반화 시킬 수 있는 단순한 컴포넌트가 아니라면 말이죠.

Reply:
말씀해주신대로 변경 했습니다 👍